app-server/migrations/src_migrations/main/scala/1.scala

```diff
@@ -0,0 +1,15 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
```

Comment:
Since the filenames are not really descriptive, I think it would be good to add some docstrings/comments for each migration about what it is doing/what it is for.